dot/parachain/availability-store/messages.go

```diff
@@ -79,9 +78,13 @@ type StoreChunk struct {
 // StoreAvailableData computes and checks the erasure root of `AvailableData`
 // before storing its chunks in the AV store.
 type StoreAvailableData struct {
-	CandidateHash       parachaintypes.CandidateHash
-	NumValidators       uint32
-	AvailableData       AvailableData
+	// A hash of the candidate this `ASMStoreAvailableData` belongs to.
+	CandidateHash parachaintypes.CandidateHash
+	// The number of validators in the session.
+	NumValidators uint
```

Comment:
why?
https://github.com/paritytech/polkadot-sdk/blob/aa68ea58f389c2aa4eefab4bf7bc7b787dd56580/polkadot/node/subsystem-types/src/messages.rs#L536

Reply:
Because this variable is used to call `erasure.ObtainChunks` which accepts a `uint`. I've update these to `uint32`, and casting to `uint` when calling `erasure.ObtainChunks`